.circleci/config.yml

```diff
@@ -158,6 +158,8 @@ jobs:
             mkdir test-jars/
             cp java/test-libs/*.jar test-jars/
             cp build/java/rocksdbjni_test_classes.jar test-jars/
+            rm test-jars/*junit-4*
```

Comment:
I think we could just remove the JUnit 4 JAR from the CMakeLists.txt?

Reply:
well, these guys have their own s3 bucket from where they download java deps in the CMake.
The Right way is obviously to put junit5 in there, ... so i just chose the [worst](https://www.jwz.org/doc/worse-is-better.html) way (-: